### PR TITLE
For EmberTimes No. 114: EmberFest Writeup

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-09-06-the-ember-times-issue-114.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 ---
 
-<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+Hiya Emberistas! 🐹
+
+This week:
 Read the Ember Octane Update for Modern Ember Tutorial 🎁,
 Participate in the discussion of deprecating eventing 💬,
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Join the Ember crowd at EmberFest 2019 🐹🎉,
```
Comment (ijlee2) [inline source/2019-09-06-the-ember-times-issue-114.md:14]: Awesome combo of emojis 💯!

File: source/2019-09-06-the-ember-times-issue-114.md [hunk 2]
```diff
@@ -66,17 +69,20 @@ This tutorial provides an intro to:
...
 
 Try out Ember Octane today which [@CodingItWrong](https://github.com/CodingItWrong) calls the “simplest, most straightforward, most productive frontend framework!”
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Emberistas from Europe and Beyond: Get Ready for EmberFest 2019 🐹🎉](https://emberfest.eu/)
+
+If you're using Ember, you're based in Europe or you're planning your next trip there soon, be sure to already mark **October 17th and 18th** in your calendar and join hundreds of other Emberistas for Europe's biggest Ember conference: **EmberFest 2019** will take place in Copenhagen, Denmark and feature **16+ talks** by community members such as [Godfrey Chan (@chancancode)](https://github.com/chancancode), [Melanie Sumner (@MelSumner)](https://github.com/MelSumner), [Edward Faulkner (@ef4)](https://github.com/ef4) and [Chantal Broeren (@cbroeren)](https://github.com/cbroeren)! If you're looking for updates on the latest and greatest in Ember and Glimmer this is the place to be.
```
Comment (ijlee2) [inline source/2019-09-06-the-ember-times-issue-114.md]:
```suggestion
If you're using Ember, you're based in Europe or you're planning your next trip there soon, be sure to mark **October 17th and 18th** on your calendar and join hundreds of other Emberistas for Europe's biggest Ember conference. **EmberFest 2019** will take place in Copenhagen, Denmark and feature **16+ talks** by community members such as [Godfrey Chan (@chancancode)](https://github.com/chancancode), [Melanie Sumner (@MelSumner)](https://github.com/MelSumner), [Edward Faulkner (@ef4)](https://github.com/ef4) and [Chantal Broeren (@cbroeren)](https://github.com/cbroeren)! If you're looking for updates on the latest and greatest in Ember and Glimmer, this is the place to be.
```

File: source/2019-09-06-the-ember-times-issue-114.md [hunk 3]
```diff
@@ -66,17 +69,20 @@ This tutorial provides an intro to:
...
+
+You can check out the [full schedule on the conference website](https://emberfest.eu/schedule/) to get an idea of the topics talked about. You can also look forward to tasty and culturally intriguing city tours on the day before the conference on top of that. Be sure to [secure your ticket before they sell out](https://emberfest.eu/) and tell your colleagues and Ember friends!
```
Comment (ijlee2) [inline source/2019-09-06-the-ember-times-issue-114.md]:
```suggestion
You can check out the [full schedule on the conference website](https://emberfest.eu/schedule/) to get an idea of the topics talked about. You can also look forward to tasty and culturally intriguing city tours on October 16th, the day before the conference. Be sure to [secure your ticket before they sell out](https://emberfest.eu/) and tell your colleagues and Ember friends!
```

File: source/2019-09-06-the-ember-times-issue-114.md [hunk 4]
```diff
@@ -66,17 +69,20 @@ This tutorial provides an intro to:
...
+
+Follow [@emberfest on Twitter](https://twitter.com/emberfest) to stay up to date with the latest news about the conference. Or subscribe to the [official mailing list](https://emberfest.eu/#mc_embed_signup).
```
Comment (ijlee2) [inline source/2019-09-06-the-ember-times-issue-114.md]: Maybe combine the next two paragraphs?
```suggestion
Follow [@emberfest on Twitter](https://twitter.com/emberfest) or subscribe to the [official mailing list](https://emberfest.eu/#mc_embed_signup) to stay up to date. See you at [EmberFest 2019](https://emberfest.eu/) soon!
```

**Review by ijlee2 — APPROVED** [review]
Thanks for writing about EmberFest! You reminded me to get the city tour ticket (food ❤️).

File: source/2019-09-06-the-ember-times-issue-114.md [hunk 5]
```diff
@@ -66,17 +69,20 @@ This tutorial provides an intro to:
...
+
+You can check out the [full schedule on the conference website](https://emberfest.eu/schedule/) to get an idea of the topics talked about. You can also look forward to tasty and culturally intriguing city tours on October 16th, the day before the conference. Be sure to [secure your ticket before they sell out](https://emberfest.eu/) and tell your colleagues and Ember friends!
+
+Follow [@emberfest on Twitter](https://twitter.com/emberfest) or subscribe to the [official mailing list](https://emberfest.eu/#mc_embed_signup) to stay up to date. See you at [EmberFest 2019](https://emberfest.eu/) soon!
+
+And see you at [EmberFest 2019](https://emberfest.eu/)!
```
Comment (jayjayjpg) [inline source/2019-09-06-the-ember-times-issue-114.md]:
```suggestion
```

**Pull request merged**